src/MessageContainer.tsx

```diff
@@ -361,7 +367,7 @@ export default class MessageContainer<
           : null}
         <FlatList
           ref={this.props.forwardRef}
-          extraData={this.props.extraData}
+          extraData={[this.props.extraData, this.props.isTyping]}
```

Comment:
Why?

Reply:
best practice is for it to be an array. you want me to keep it an obj?

Reply:
are you asking why isTyping needs to be inside of extraData? it's because the FlatList needs to rerender the footer where the typing animation lives.

Reply:
OK :)